tests/experimental/vector/test_vector_make.py

```diff
@@ -0,0 +1,59 @@
+import pytest
```

Comment:
Move to `tests/envs/registration`

Reply:
Experimental tests mirror the structure of original vector tests, so I'll keep it that way for now, we can do relocations later (it's tests, so low priority)